evap/rewards/tests/test_tools.py

```diff
@@ -45,6 +45,20 @@ def test_semester_activated_not_all_courses(self):
         self.form.submit()
         self.assertEqual(0, reward_points_of_user(self.student))
 
+    def test_removes_from_course(self):
+        SemesterActivation.objects.create(semester=self.course.semester, is_active=True)
+        mommy.make(Course, semester=self.course.semester, participants=[self.student])
+        self.form.submit()
+        self.student.courses_participating_in.set([self.course])
```

Comment:
is something like `self.student.courses_participating_in.remove(self.course)` possible here as well? i think that would make clearer what's actually happening